FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.DataClient/Api/Dicom/DicomApiConstants.cs

```diff
@@ -47,8 +47,13 @@ public class DicomApiConstants
         public const string LatestKey = "latest";
 
         /// <summary>
-        /// The managed identity url to get access token for DICOM service.
+        /// The managed identity audiences to get access token for DICOM service.
         /// </summary>
-        public const string DicomResourceUrl = "https://dicom.healthcareapis.azure.com";
+        public static readonly Dictionary<ServerEnvironmentGroup, string> DicomServerAudienceMap = new Dictionary<ServerEnvironmentGroup, string>()
+        {
+            { ServerEnvironmentGroup.PROD, "https://dicom.healthcareapis.azure.com" },
+            { ServerEnvironmentGroup.TEST, "https://dicom.healthcareapis.azure-test.net" },
+            { ServerEnvironmentGroup.LOCAL, "https://dicom.healthcareapis.azure-test.net" },
```

Comment:
Are the values for TEST and LOCAL environment the same?

Reply:
If so, maybe we can just keep TEST here?

Reply:
Remove this property and directly set up audience in appsetting configuration.